WirelessGamingReceiver/WirelessGamingReceiver.cpp

```diff
@@ -362,13 +359,13 @@ void WirelessGamingReceiver::ReleaseAll(void)
         }
         if (connections[i].controllerIn != NULL)
         {
-            connections[i].controllerIn->Abort();
+            connections[i].controllerIn->abort(kAbortOptionNone, kIOReturnAborted, this);
```

Comment:
I don't know where you got `kAbortOptionNone`, but that doesn't seem to be in the macOS' current SDKs.